### PR TITLE
Unpack data translator packages returned from Envoy app_info

### PR DESCRIPTION
**Pull request by timotree3**
Hopefully fixes https://github.com/Holo-Host/holo-envoy/issues/63

### CODE REVIEW

**Comment by timotree3** [comment]
Actually this fails tests because the tests expect Chaperone to output data translator format as well

**Comment by alastairong** [comment]
@timotree3 should we ignore for now?

**Comment by timotree3** [comment]
not ready for review until tests pass

**Comment by timotree3** [comment]
ready for review

File: src/index.ts [hunk 1]
```diff
@@ -999,25 +999,22 @@ class Chaperone {
...
       // hhdt does not wrap the response in a payload obj when returning the data directly from the conductor
-      return appInfo.cell_data;
+      return this.appWebsocket.appInfo({ installed_app_id });
     } else {
-      return appInfo.payload.cell_data;
+      const pkg = await this.conn.call(ENVOY_APP_INFO, { installed_app_id });
+      if (pkg.type === "success") {
+        return pkg.payload;
+      } else {
+        throw new Error(`Error while calling envoy app_info: ${JSON.stringify(pkg)}`);
```
Comment (robbiecarlton) [inline src/index.ts:1010]: data translator packages have a `toString` method that you should probably use here.

**Review by robbiecarlton — APPROVED** [review]
One non blocking request

**Review by zo-el — APPROVED** [review]

**Pull request merged**